skeleton/docker-entrypoint.sh

```diff
@@ -27,6 +27,8 @@ fi
 [ -z ${DEFAULT_ZPUBLISHER_ENCODING+x} ] && export DEFAULT_ZPUBLISHER_ENCODING=utf-8
 [ -z ${COMPILE_MO_FILES+x} ] && export COMPILE_MO_FILES=true
 [ -z ${DEBUG_MODE+x} ] && export DEBUG_MODE=off
+[ -z ${ZOPE_DOS_PROTECTION+x} ] && export ZOPE_DOS_PROTECTION=4MB
```

Comment:
I would call this ZOPE_FORM_MEMORY_LIMIT. It is not the only setting related to DOS protection that Zope now supports.

Reply:
@davisagli Solved